Delete pt2e flow from pytorch/pytorch since it's migrated to torchao

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #169151

Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc @albanD @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben @jataylo @Lucaskabela

jerryzh168 committed:
Delete pt2e flow from pytorch/pytorch since it's migrated to torchao

Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented:
@jerryzh168 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D87958849).

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]

jerryzh168 commented:
@jerryzh168 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D87958849).

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

albanD commented on test/inductor/test_cpu_select_algorithm.py:
```diff
@@ -1323,6 +1322,9 @@ def forward(self, arg152_1):
         ),
     )
     @skipIfWindows(msg="Windows don't support quantize.")
+    @unittest.skip(
```
Can we delete tests instead of always skipping?  The git history will keep all the details of these tests for when the migration is being done.

albanD reviewed (commented):
Very nice!

anijain2305 commented on torch/_dynamo/variables/nn_module.py:
```diff
@@ -506,9 +506,10 @@ def call_function(
-                and not isinstance(
-                    mod, torch.ao.quantization.pt2e.export_utils._WrapperModule
-                )
+                # TODO: how to do this for torchao pt2e code?
```
cc @tugsbayasgalan - does this remind you of anything?

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 commented:
@jerryzh168 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D87958849).

jerryzh168 committed:
Update on "Delete pt2e flow from pytorch/pytorch since it's migrated to torchao"


Summary:

Deleting pt2e flow for 2.10 as mentioned in https://docs.pytorch.org/docs/stable/quantization.html

Other flows still have internal users, we'll revisit later.

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D87958849](https://our.internmc.facebook.com/intern/diff/D87958849)

cc albanD voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy kadeng muchulee8 amjames chauhang aakhundov coconutruben jataylo Lucaskabela

[ghstack-poisoned]

jerryzh168 commented:
@jerryzh168 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D87958849).
